src/FakeItEasy/An.of.T.cs

```diff
@@ -0,0 +1,44 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <summary>
+    /// Provides an API entry point for constraining arguments of fake object calls.
+    /// </summary>
+    /// <typeparam name="T">The type of argument to validate.</typeparam>
+    /// <remarks>This is a synonym of <see cref="A{T}"/>, to enable proper grammar when the type of T starts with a vowel.</remarks>
+    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = nameof(A), Justification = "It is spelled correctly.")]

```

Comment:
`nameof(An)`?

Reply:
Or maybe the suppression can be removed?

Reply:
Oops. Copy paste mistake.
I *think* the suppression could be removed; at least Visual Studio thinks so.